### DISCUSSION TITLE
Issue Accessing the AP WiFi setup page

### DISCUSSION THREAD

**Discussion by maisyk** [HomeSpan/HomeSpan#609] [Q&A] [answered]
So I used `homeSpan.enableAutoStartAP();` it started the AP and I was able to connect to it. But it doesn't show up the actual browser window for me to connect the ESP32 board to the WiFi itself. I have tried from my Android, iPhone and my Mac, but had the same issue. 

Just to add in the Serial Monitor I see the following:

```
*** WIFI CREDENTIALS DATA NOT FOUND.  AUTO-START OF ACCESS POINT ENABLED...

*** Starting Access Point: HomeSpan-Setup / homespan

Scanning for Networks...

  1) XXXXX
  2) XXXXX
  3) ....

Ready.

*** Access Point Terminated.  Restarting...
```

Not sure if that message regarding the AP being terminated means anything, but I can still see it on my device's WiFi list.

Any suggestions on this one?

>**Note:** I initially commented on [this](https://github.com/HomeSpan/HomeSpan/discussions/148) discussion but later realized this doesn't quite match up with the topic being discussed there (and that discussion has been in an _Answered_ state since 2021).

___

## Update/ Notes:
After a bit more messing around I have found that the issue may potentially be coming from the control button I have. If I comment out the call to`homeSpan.setControlPin()` the AP + WiFi config web page works fine.

**Comment by HomeSpan** [comment, marked as answer]
The message you are receiving will occur if you try to cancel the AP by pressing down the control button while it is running.  I just tried with a control button on my side and it appears to be working correctly.  How have you configured your control button?  Is it using a valid ESP32 pin that does not conflict with anything else?

**Reply by maisyk** [reply]
Hey @HomeSpan, thank's for the quick response. I was just about to write up a comment regarding this. 

Seems the issue was indeed caused by the control button (on a hardware level). Initially, I thought I must have wired it wrong whipped out the multimeter and all was well but it seemed with slight vibration the button was actually going into a closed state. 

Just swapped out the button for a new one and seems to work spot on. :) 

I suppose this discussion can now be closed, again thank you for your quick response! 

TLDR; Issue was caused by a bad button that would go into a closed state upon being subjected to any vibration.

**Reply by HomeSpan** [reply]
Thanks for the update - glad you found the issue and things are working.
